KsApi/models/Reward.swift

```diff
@@ -25,14 +25,28 @@ public struct Reward {
 
   public struct Shipping {
     public let enabled: Bool
+    public let location: Location?
     public let preference: Preference?
```

Comment:
This is deprecated but I think we should only remove it when we remove `DeprecatedRewardPledgeViewController`.